Updated contour list recycler view with table view.

chaneylc opened the pull request.

chaneylc committed:
Updated contour list recycler view with table view.

chaneylc committed:
Finished table view

trife commented:
This looks great!

- Vertically center the count text (currently slightly above the other text elements in each row)
- Horizontally center the checkmark (currently skewed left)
- Can we selectively highlight columns? `Counted` doesn't need to be highlighted but it would be nice to use the column title to select/deselect all of the elements of the list.
- Replace `0.0` with `N/A` for group measurements

chaneylc committed:
Removed highlighting
Fixed bug where total shows 1 count if 0 are selected.
Improved contour list UI
Replaced 0.0 with N/A for group measurements.
Clicking counted header will select/deselect all.

trife committed:
Merge branch 'dev' into issue32_v2.1

trife committed:
Updated changelog

trife commented:
This looks great!!
